terraform/templates/dashboards/logging.json

```diff
@@ -239,6 +1364,6 @@
   },
   "timezone": "",
   "title": "Logging",
-  "uid": "JMhAZhVGk",
-  "version": 3
-}
+  "uid": "4Na_dsIMk",
+  "version": 4
+}
```

Comment:
CI might complain about the missing newline here.

Reply:
it 100% will.  good catch on it.